### PR TITLE
Feature/fix inlinestyles

### PR DESCRIPTION
**Pull request by faebeee**
There is an error when an inline style has the `rel=stylesheet` property. 
Added a check to see if there is a property named `href` on the tag.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -55,7 +55,11 @@ module.exports = function(url, options){
     result.pageTitle = $('head > title').text();
 
     $('[rel=stylesheet]').each(function() {
-      var link = $(this).attr('href');
+        var link = $(this).attr('href');
+        if (!(typeof link !== typeof undefined && link !== false)) {
```
Comment (johnotander) [inline index.js]: Mind pulling in [`is-present`](https://www.npmjs.com/package/is-present) and making the logic `if`/`else`?

```js
if (isPresent(link)) {
  result.links.push(createLink(link, url));
} else {
  result.styles.push(stripHtmlComments($(this).text()));
}
```

**Comment by johnotander** [comment]
Nice catch!

File: index.js (hunk 1 above)
Comment (faebeee) [reply index.js]: Yea sure

**Comment by johnotander** [comment]
Awesome, thanks!

**Pull request merged**